.coveralls.yml

```diff
@@ -1,2 +0,0 @@
-service_name: travis-ci
-repo_token: O7N8dGjWu6Z2BEcBkkfggKgwQecw5lvqQ
```

Comment:
Note, this is not meant to be public. You should regenerate it from the coveralls.io repo settings!

Reply:
Whoops, you're quite right 😳